wishlist.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/630f3095fb.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://practical-poitras-d410a0.netlify.app/packages/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quintessential&family=Roboto:ital,wght@0,300;1,400&family=Zen+Kurenaido&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar-title">
+            Astra
+        </div>
+        <div>
+            <a href="index.html" class="navbar-link mr-s">Home</a>
+            <a href="product.html" class="navbar-link">product</a>
+        </div>
+        <div class="navbar-items">
+            <div>
+                <ul class="navbar-list">
+                    <li class="navbar-item navbar-icon">
+                        <a href="cart.html" class="navbar-link">
+                            <span class="btn-icon"><i class="fas fa-shopping-cart"></i>
+                            </span>
+                        </a>
+                    </li>
+                    <li class="navbar-item navbar-icon">
+                        <a href="wishlist.html" class="navbar-link">
+                            <i class="fas fa-heart"></i></a>
+                    </li>
+                </ul>
+            </div>
+            <a href="login.html" class="btn btn-primary">Login</a>
+        </div>
+    </nav>
+    <div class="container">
+        <div class="text-secondary text-center my-m">My Wishlist</div>
+        <div class="grid grid-1-4">
+            <div>
+                <div class="card">
+                    <div class="card-vertical-sm">
+                        <div class="card-img card-img-3">
+
+                        </div>
+                        <div class="card-head text-center">
+                            <h3 class="card-title">Vogue</h3>
+                            <span>₹5000</span>
+                        </div>
+                        <div class="card-links">
+                            <button class="btn btn-primary btn-large">Move to cart</button>
+                        </div>
+                        <span class="badge-quinary"><i class="far fa-heart"></i></span>
```

Comment:
Here, have the heart icon in red color itself as it's already in the wishlist right?